homeassistant/exceptions.py

```diff
@@ -67,7 +91,7 @@ class NoEntitySpecifiedError(HomeAssistantError):
 class TemplateError(HomeAssistantError):
     """Error during template rendering."""
 
-    def __init__(self, exception: Exception | str) -> None:
+    def __init__(self, exception: Exception | str | None = None) -> None:
```

Comment:
Why is this changed?

Reply:
I changed it so that translations can be supported, but will revert this in the current scope of the PR.